### ISSUE TITLE
Downloader module move race condition

### ISSUE THREAD

**Issue by nicholasjpaterno** [trufflesuite/ganache#1524]
The downloader module has a race condition.  It only occurs on the first run when modules are downloading and moved.  It appears that creating the workspace will attempt to move the module again but cannot find the directory.

**Comment by nicholasjpaterno** [comment]
Example Error:
```
Error: ENOENT: no such file or directory, open '/home/cashlion/.config/Electron/extras/corda-tools-network-bootstrapper-4.3.jar.downloading'
```

**Comment by nicholasjpaterno** [comment]
Easiest way to recreate is to delete the `.config/Electron/extras` dir and restart ganach.

Click: Download Corda then immediately click quickstart.

**Comment by nicholasjpaterno** [comment]
Resolved by 7e3f13f
